src/main/java/org/fundacionjala/at13/spaceinvaders/GameBoard.java

```diff
@@ -46,27 +63,68 @@ public void init() {
         }
         spaceShip();
         addKeyListener(this);
+        start();
     }
 
     /**
      * Method to initialize and show the spaceShip.
      */
     public void spaceShip() {
         spaceship = new Spaceship(SCALE_WIDTH / 2, SCALE_HEIGHT - 1, 0, SCALE_WIDTH);
+        bullet = spaceship.shoot();
+        refresh();
+    }
+
+    /**
+     * Method to initialize timer.
+     */
+    public void start() {
+        TimerTask task = new TimerTask() {
+            @Override
+            public void run() {
+                if (tic % VELOCITY_GROUP_ALIEN == 0) {
+
```

Comment:
maybe remove this line break in the code

Reply:
Already solved 😁